### PR TITLE
Improved testing select_contact

### LINKED ISSUE #1520: Tests for select_contact.dart
- Please coordinate **issue assignment** and **PR reviews** with the contributors listed in this issue https://github.com/PalisadoesFoundation/talawa/issues/359

The Talawa code base needs to be 100% reliable. This means we need to have 100% test code coverage.

Tests need to be written for file `lib/views/after_auth_screens/chat/select_contact.dart`

- When complete, all methods, classes and/or functions in the file will need to be tested. 
- Our `test/` folder has the same structure as the `lib` folder. Place your test file in the equivalent folder under `test/`. You may need to create the appropriate directory structure to do this.
- [Current test code coverage for this file can be found here](https://app.codecov.io/gh/PalisadoesFoundation/talawa?search=&displayType=list)

### IMPORTANT:
Please refer to this issue on how to implement these tests correctly:

- https://github.com/PalisadoesFoundation/talawa/issues/1217

### PR Acceptance Criteria

- When complete this file must show **100%** coverage when merged into the code base. This will be clearly visible when you submit your PR.
- The PR will show a report for the code coverage for the file you have added. You can use that as a guide.
- You can verify your own code coverage by creating an account at [Codecov.io](https://codecov.io)
- No functionality must be removed to achieve this goal.

### PR DESCRIPTION
**Pull request by Kraken005**
**What kind of change does this PR introduce?**

This PR improves the testing of select_contact.dart file.

**Issue Number:**
#1520 

Fixes #1520 
**Did you add tests for your changes?**

Yes. I added test for title display.

**Summary**

A new widget testing for title display. It would overall increase the testing code coverage.

**Does this PR introduce a breaking change?**

No

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa/blob/master/CONTRIBUTING.md)?**

Yes

### CODE REVIEW

**Comment by palisadoes** [comment]
Please fix the failing tests. Our test code coverage system will fail if either of these two conditions occur:

- The overall code coverage drops below the target threshold of the repository
- Any file in the pull request has code coverage levels below the repository threshold

We do this to ensure the overall reliability of the code base is maintained.

**Comment by Cioppolo14** [comment]
@Kraken005 Please fix the failing tests. Our test code coverage system will fail if either of these two conditions occur:

The overall code coverage drops below the target threshold of the repository
Any file in the pull request has code coverage levels below the repository threshold

We do this to ensure the overall reliability of the code base is maintained.

**Review by noman2002 — CHANGES_REQUESTED** [review]
fix the formatting and why there is change in pubspec.lock ?

**Comment by noman2002** [comment]
@Kraken005 Any progress on this one ??

File: .github/workflows/codecov.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        uses: subosito/flutter-action@v1
+        with:
+          flutter-version: '3.7.1' 
+      - name: Install dependencies
+        run: flutter pub get
+      - name: Run tests with coverage
+        run: flutter test --coverage
+      - name: Upload coverage reports to Codecov
+        uses: codecov/codecov-action@v3
+
```
Comment (noman2002) [inline .github/workflows/codecov.yml:26]: Why a new workflow is added?

**Comment by Kraken005** [comment]
The code base testing failed at /home/runner/work/talawa/talawa/test/widget_tests/widgets/task_schedule_test.dart.

**Comment by noman2002** [comment]
Please add documentation for the changes you have made

**Comment by palisadoes** [comment]
Please fix the failing tests. Our test code coverage system will fail if either of these two conditions occur:

1) The overall code coverage drops below the target threshold of the repository
2) Any file in the pull request has code coverage levels below the repository threshold

We do this to ensure the overall reliability of the code base is maintained. This is a prerequisite for getting your PR approved. Assigned reviewers regularly review the PR queue and tend to focus on PRs that are passing.

**Review by noman2002 — CHANGES_REQUESTED** [review]
Please fix the failing test.

**Comment by palisadoes** [comment]
closing due to a inactivity

**Pull request closed without merging**